src/fulfiller_identity_proxy.js

```diff
@@ -26,25 +25,30 @@ class FulfillerIdentityProxy {
     return new Promise((resolve, reject) => {
       self.xray.captureAsyncFunc('FulfillerIdentity.getFulfillerById', function (subsegment) {
         self.authenticator.getAuthorization().then((authorization) => {
+          const axiosInstance = axios.create({
```

Comment:
If we are going to create an instance, can we "cache" it by creating it globally? Do we need a dedicated instance or we can just use the default one?

Reply:
we don't need a dedicated instance and can just use the default.